modules/social_features/social_core/social_core.install

```diff
@@ -1642,3 +1642,17 @@ function social_core_update_11405(): string {
   // Output logged messages to related channel of update execution.
   return $update_helper->logger()->output();
 }
+
+/**
+ * Ajust popovers socialblue theme configs.
+ */
+function social_core_update_11406(): string {
```

Comment:
@volodymyr-sydor Since we add this to the milestone `11.8.x`, please change number to start with `11801` just in case we might have other updates for previous versions.

Reply:
done